argus.js

```diff
@@ -12,8 +12,12 @@ const ArgusJS = function(token) {
     }, false);
 
     return {
-        token: () => currentToken,
-        get: (url) => fetch(url, { "method": "GET", "headers": { "Authorization": "Token " + currentToken } })
+        get: (url) => fetch(`${mdrUrl}${url}`, { "method": "GET", "headers": { "Authorization": "Bearer " + currentToken.access } }),
+        post: (url, data) => fetch(`${mdrUrl}${url}`, { "method": "POST", "headers": { "Authorization": "Bearer " + currentToken.access }, body: JSON.stringify(data) }),
```

Comment:
For all the http request types that have a body you need to explicitly define the content type in the headers as the following  'Content-Type': 'application/json; charset=UTF-8'